networking/configuring-node-port-service-range.adoc

```diff
@@ -6,15 +6,22 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can expand the available node port range. If your cluster uses of a large number of node ports, you might need to increase the number of available ports.
+During cluster installation, you can set the node port range to a range meets the needs of your cluster. Only a cluster administrator can change the range by expanding the range as a postinstallation task. If your cluster uses a large number of node ports, consider increasing the number of available ports for your cluster.
 
-The default port range is `30000-32767`. You can never reduce the port range, even if you first expand it beyond the default range.
+[IMPORTANT]
+====
+Before you expand a node port range, consider that Red{nbsp}Hat has not performed testing outside the default port range of `30000-32768`. For ranges outside the default port range, ensure that you test to verify the expanding node port range does not impact your cluster. If you expanded the range and a port allocation issue occurs, create a new cluster and set the required range for it. 
+====
 
-[id="configuring-node-port-service-range-prerequisites"]
-== Prerequisites
+If you do not set a node port range during cluster installation, the default range of `30000-32768` applies to your cluster. In this situation, you can expand the range on either side, but you must preserve `30000-32768` within your new port range.
 
-- Your cluster infrastructure must allow access to the ports that you specify within the expanded range. For example, if you expand the node port range to `30000-32900`, the inclusive port range of `32768-32900` must be allowed by your firewall or packet filtering configuration.
+[IMPORTANT]
+====
+If you expand the node port range and {oc-first} stops working because the OpenShift API server
```

Comment:
If you expand the node port range and {oc-first} stops working due to a port conflict with the OpenShift API server, you must create a new cluster.  
Ensure that the new node port range does not overlap with any ports already in use by host processes or pods configured with host networking.

Reply:
:heavy_check_mark: